hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -519,6 +519,20 @@ def all(cls) -> List["HealthCheck"]:
     This check requires the :ref:`Hypothesis pytest plugin<pytest-plugin>`,
     which is enabled by default when running Hypothesis inside pytest."""
 
+    differing_test_runners = 10
+    """Checks if :func:`@given <hypothesis.given>` has been applied to a test
+    which is executed by different test runners. If the test runners have different
```

Comment:
I think the "test runners" terminology is a weird historical artefact of the Hypothesis internals, so we should probably call this an "executor" and link to https://hypothesis.readthedocs.io/en/latest/details.html#custom-function-execution

Maybe also try mentioning that 'If your test function is defined as a method on a class, that class will be your executor, and subclasses executing an inherited test is a common way for things to go wrong' - or something to similar effect.